scripts/examples/python/setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, find_packages
+
+setup(name='XenAPI',
+      version='1.2',
+      description='XenAPI SDK, for communication with Citrix Hypervisor.',
+      author='Citrix Systems, Inc.',
+      author_email='discard@citrix.com',
```

Comment:
Should we use xen-api@lists.xenproject.org instead?

Reply:
It looks a bit strange since in pypi: the author is Citrix, but the mailto link points to the mail list (on the left-hand side) https://test.pypi.org/project/XenAPI/2.14/